website/docs/r/loadbalancer_nat_rule.html.markdown

```diff
@@ -8,7 +8,7 @@ description: |-
 
 # azurerm_lb_nat_rule
 
-Manages a Load Balancer NAT Rule.
+Manages a Load Balancer NAT Rule. This resource is intended to use with network interfaces of virtual machines.
```

Comment:
As above could we revert this
```suggestion
Manages a Load Balancer NAT Rule.
```

Reply:
Thanks for reviewing.
I might be wrong, but I've made those note(s) because in _ARM Template reference_ (**API version 2019-04-01**) and _REST API reference_ (**API version 2019-07-01**) regarding creating\updating Load Balancer resource in Azure there are some clarifications about using\referencing NAT Pools\NAT Rules:
https://docs.microsoft.com/en-us/azure/templates/microsoft.network/2019-04-01/loadbalancers#LoadBalancerPropertiesFormat
https://docs.microsoft.com/en-us/rest/api/load-balancer/loadbalancers/createorupdate#request-body

Reply:
as above,
```suggestion
Manages a Load Balancer NAT Rule. 

-> **NOTE:** This resource cannot be used with with virtual machine scale sets, instead use the `azurerm_lb_nat_pool` resource.
```